MikuMikuWorld/DefaultLanguage.h

```diff
@@ -29,6 +29,7 @@ namespace MikuMikuWorld
 		{ "create_auto_save", "Create Auto Save" },
 		{ "help", "Help" },
 		{ "about", "About" },
+		{ "layer", "Layer" },
```

Comment:
What is this translation for?

Reply:
It was not capitalized previously, unlike everything else

Reply:
Makes sense, thanks!